template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/CmHostGroupRoleConfigProviderProcessorTest.java

```diff
@@ -226,22 +226,25 @@ public void testGenerateConfigsWhenHdfsJournalNodeNotNull() {
         setup("input/cb5660.bp", Builder.builder().withHostgroupViews(Set.of(master1, master2, worker, compute, quorum)));
 
         Map<String, Map<String, List<ApiClusterTemplateConfig>>> actual = underTest.generateConfigs(templateProcessor, templatePreparator);
-        assertNull(actual.get("hdfs-JOURNALNODE-BASE"));
+        assertNotNull(actual.get("hdfs-JOURNALNODE-BASE"));
     }
 
     @Test
-    public void testGenerateConfigsWhenHdfsJournalNodeIsNotNull() {
-        List<SharedComponent> sharedComponents = List.of();
+    public void testProcessWhenSharedConfigThenAddVolumeConfig() {
```

Comment:
I do not see such test which especially covers the ZK case

Reply:
done